src/api/queries/getMetricQueries.js

```diff
@@ -0,0 +1,34 @@
+ApiClient_.prototype.fetchGetMetric = function (metric, slug, from, to, timeBoundSuffix) {
+  var timeBound = ['', '1d', '7d', '30d', '60d', '90d', '180d', '365d', '2y', '3y', '5y', '10y']
+  var timeBoundMetrics = [
+    'mean_realized_price_usd',
+    'mvrv_usd',
+    'circulation',
+    'realized_value_usd'
+  ]
+
+  if (timeBoundSuffix !== '') {
+    if ((timeBound.indexOf(timeBoundSuffix) !== -1) && (timeBoundMetrics.indexOf(metric) !== -1)) {
+      var reworkedTimeBound = '_' + timeBoundSuffix
+    } else {
+      throw Error
+    }
+  } else {
+    reworkedTimeBound = ''
+  }
```

Comment:
you can simplify this by:
```javascript
if (timeBoundSuffix != '') {
  if ((timeBound.indexOf(timeBoundSuffix) !== -1) && (timeBoundMetrics.indexOf(metric) !== -1)) {
    metric = metric + '_' + timeBoundSuffix
  } else {
    throw Error
  }
}
```

and just use `metric` below, without the the `+ reworkedTimeBound`, which can be confusing (as not all metrics are timebound and at first glance it seems you're always appending)

Reply:
Also, can we throw a meaningful error here? With description why we throw?

Reply:
I tried it without the new variable, but doesn't work, because eslint doesn't allow me to modify the function argument. The error should be more meaningful now, will add a description in a bit.